src/main/java/net/sacredlabyrinth/Phaed/PreciousStones/PreciousStones.java

```diff
@@ -190,6 +186,9 @@ private void registerEvents() {
         getServer().getPluginManager().registerEvents(worldListener, this);
         getServer().getPluginManager().registerEvents(inventoryListener, this);
 
+        if (permissionsManager.hasWorldEdit() && !settingsManager.isDisableWorldEditHook()) {
+            getPlugin(WorldEditPlugin.class).getWorldEdit().getEventBus().register(new WorldEditListener());
+        }
         if (permissionsManager.hasMcMMO()) {
```

Comment:
I think if you move this code to be inside of WorldEditListener (like a static register method) this would be OK. But we can't access any WorldEdit classes in any code that could get hit before we know for sure the plugin is present.